### ISSUE TITLE
Choose a Suitable License

### ISSUE THREAD

**Issue by zth96** [UBC-MDS/TidyverseToPandas#69]
Hello Team. I created this issue to discuss about whether we should change to a new license or maintain the MIT license. 

I think it really depends on the purpose of our project. Given the nature of our package, which is a software tool aimed at facilitating code translation between two programming environments, I think either the MIT License or GNU General Public License (GPL) might be more aligned with typical software licensing practices. Plus, I don't think our package will be used and shared commercially so I think no commercial license to be used here. 

If we require derivative works to be open source, distributors to make our source code available, and focus on building open-source ecosystem within the community, GNU General Public License (GPL) license might be our best choice. If we just want minimal restrictions that only requires preserving the license and copyright notice, we can just keep the MIT license. 

This is just my personal opinion, please contribute your thoughts on how we limit the use of our package.

**Comment by farrandi** [comment]
Thanks for doing some research on it!

I agree with your take. We should stick to the MIT license. Mainly because it is simple.

Thanks for checking into this!

I agree with your choice of the MIT license because it's straightforward.

Although it has some limitations and allows commercial use, these aspects are fine for us. Our main goal for this project is to assist people moving from R to Python, so we want them to use our code freely for whatever they need.

**Comment by LilyTao0531** [comment]
Yes I think we may want to use MIT license. As what you guys have mentioned, distributors not expected, and the nature of our project (wrapping the Pandas function with the format of a R Tidyverse function), indicate MIT is the most suitable license that we want to go with. The non-commercial purpose of this project does not conflict with MIT license.

**Comment by 786213750** [comment]
Thank you all for your inputs. I agree with the choice of the MIT license for our project. The primary goal of this project is to encourage the use of Pandas DataFrames, especially for those accustomed to dplyr in R. We aim to maintain this as an open-source and non-profit initiative, and the MIT license aligns perfectly with these objectives. Additionally, the MIT License affords robust safeguards for us as developers, notably by providing liability protection and explicitly stating a no-warranty clause, thereby reinforcing the security of our collaborative endeavors.

**Comment by farrandi** [comment]
sounds good! 

I believe we are all in agreement of the MIT license. 

No action items because we are not changing anything. 

The issue is resolved but leaving this "open" for visibility.
